lib/streamlit/caching/memo_decorator.py

```diff
@@ -435,9 +449,8 @@ def clear(self) -> None:
     def _read_from_mem_cache(self, key: str) -> bytes:
```

Comment:
Should add some documentation explaining why these work with bytes rather than a structured type (avoid unneeded roundtripping when saving to mem a value from disk)